cstruct.opam

```diff
@@ -1,22 +1,29 @@
-opam-version: "1.2"
+opam-version: "2.0"
 maintainer:   "anil@recoil.org"
 authors:      ["Anil Madhavapeddy" "Richard Mortier" "Thomas Gazagnaire"
                "Pierre Chambart" "David Kaloper" "Jeremy Yallop" "David Scott"
                "Mindy Preston" "Thomas Leonard" ]
 homepage:     "https://github.com/mirage/ocaml-cstruct"
 license:      "ISC"
-dev-repo:     "https://github.com/mirage/ocaml-cstruct.git"
+dev-repo: "git+https://github.com/mirage/ocaml-cstruct.git"
 bug-reports:  "https://github.com/mirage/ocaml-cstruct/issues"
 tags: [ "org:mirage" "org:ocamllabs" ]
 build: [
   ["jbuilder" "subst" "-p" name "--name" name] {pinned}
   ["jbuilder" "build" "-p" name "-j" jobs]
+  ["jbuilder" "runtest" "-p" name] {with-test}
 ]
-build-test: ["jbuilder" "runtest" "-p" name]
-
 depends: [
-  "jbuilder" {build & >="1.0+beta10"}
-  "sexplib"
+  "ocaml" {>= "4.03.0"}
+  "jbuilder" {build & >= "1.0+beta10"}
+  "sexplib" {< "v0.12"}
   "alcotest" {test}
```

Comment:
This was the faulty line btw -- `{test}` should `{with-test}` in opam2. `opam package-upgrade *.opam` should automate this